### PR TITLE
resolve #37 - VOTING AMENDMENT 🎉

### LINKED ISSUE #37: Refactor Article VI, Voting
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:

This issue is for several possible refactors involving the voting processes described in Article VI.
- [x] The Simple Majority vote described in 6.C.1 describes a Relative Majority / Plurality vote, not a Simple Majority vote. Having both options (Relative Majority and Simple Majority) may be beneficial.
- [x] Combine language for Immediate and Secret Immediate Votes 6.B.3/6.B.4.
- [x] Combine "Fifty Percent", "Two Thirds", and "Three Quarters" votes.
- [x] Remove paper ballot requirements.
- [x] 6.D.1 may not be needed, depending on how votes in 6.C are described.
- [x] Rename "Alternative" vote (6.C.6) "Ranked Choice".
- [x] Remove "magic number" language in 6.C ("Below are listed four standard votes." - there are six, but better would be to remove this language entirely).
- [x] Clarify "Number of Votes Required" language. It seems strange to have it cover both the requirements for Quorum and for votes to pass.
- [ ] Language throughout the Constitution is inconsistent when describing votes' Quorum, Procedure, Number of Votes Required, and other characteristics. It would likely be best practice to define a consistent framework for discussing votes and replace current language with the new framework. 
  - Additionally, any votes should be modified as needed to reflect current practice.

### PR DESCRIPTION
**Pull request by rhochgraf21**
Check one:
- [X] Semantic Change: something about the meaning of the text is different.
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

- Definitions for Voting have been updated. "Total Number of Possible Votes" -> "Eligible Votes", and the like.
- "Alternative Vote" is no longer described as a Method of Vote as well as a means of counting votes; it is now only a means of counting votes. Also it's been renamed to "Ranked Choice".
- Imposed quorum requirements for votes that had previously been missing quorums.
- Replaced Every™ vote in the Constitution with new format.

Resolves #37

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -562,13 +559,13 @@
 
 \asubsubsubsubsection{Voting}
 On a member by member basis, this evaluation process determines if each Introductory member has successfully completed the Introductory Process requirements \ref{Expectations of an Introductory Member}.
-A Simple Majority vote with a quorum of two-thirds of the Total Number of Possible Votes is required for the evaluation to be valid.
-In order for an Active Member to be counted as a Possible Vote they must meet the requirements outlined in \ref{Expectations of Voting Members}.
+An Immediate Relative Majority Vote with two-thirds Quorum is required for the evaluation to be valid.
+In order for an Active Member to be eligible to vote during the evaluation process they must meet the requirements outlined in \ref{Expectations of Voting Members}.
```
Comment (danisaba) [inline constitution.tex:563]: This is just me being nitpicky, but "In order for" doesn't need to exist. "For an Active Member to be eligible to vote during the evaluation process...." etc etc

File: constitution.tex [hunk 2]
```diff
@@ -821,7 +818,8 @@
 	\item During an official Executive Board Vote, each dual directorship member's vote counts for one-half vote in the tallying of votes.
 		The members of the dual directorship need not vote the same way in a vote.
 	\item A member of a dual directorship may not hold any other Executive Board position.
-	\item When attendance requirements call for the dual directorship position to be present, both dual directorship members are to fulfill the requirements.
+	\item When attendance requirements call for the dual directorship position to be present, both directorship members must be present to fulfill the requirements.
+	\item Dual directorships must have both members present for to count towards Quorum in an Executive Board Vote.
```
Comment (danisaba) [inline constitution.tex:822]: is this a true statement that reflects current practice?

File: constitution.tex [hunk 3]
```diff
@@ -834,16 +832,16 @@
...
 	\item Each candidate will be given an equal amount of time before the House to present their platform of candidacy.
-	\item Ballots will then be distributed and voting will be open for a minimum of a forty-eight hour period.
+	\item Ballots will then be distributed for a Balloted Vote \ref{Balloted Vote} and voting will be open for a minimum of a forty-eight hour period.
 		The Ballots will list, in random order, all of the candidates who are qualified for a given office with a means to indicate the selection of one of the candidates.
 		In addition, an area will be provided to indicate a write-in selection of a candidate.
-	\item At the end of the voting period, the Chairperson will terminate voting by collecting the ballot box and the votes shall be counted as stated in \ref{Alternative Vote}.
-	\item The winners are determined via the process described in \ref{Alternative Vote}.
-		A quorum of fifty-percent of the number of all members eligible to vote is required for the election to be official.
+	\item At the end of the voting period, the Chairperson will terminate voting by collecting the ballot box.
```
Comment (danisaba) [inline constitution.tex:838]: We do not use a ballot box anymore. So I am not sure how to update this to reflect current practices though I feel we should

**Review by danisaba — CHANGES_REQUESTED** [review]
I think I made one nitpicky change. I also want to discuss the word "ballot" and its use. Also want to discuss dual directorships as mentioned. Anyways read my comments and prepare for sad

File: constitution.tex [hunk 4]
```diff
@@ -907,95 +905,71 @@
...
+After the appeal is presented, an Executive Board Vote is taken to determine whether or not to overturn and reevaluate the decision.
+If the vote passes, the Executive Board may discuss and make a new ruling by another Executive Board Vote.
 
 % ARTICLE VI - VOTING
 \article{Voting}
 This section outlines the different types of votes and ballots used to make House decisions and defines relevant terminology.
 \asection{Definitions}
-\asubsection{Total Number of Possible Votes}
+\asubsection{Eligible Votes}
 The sum of the number of Active members eligible to vote on the issue.
```
Comment (Sam-Cordry) [inline constitution.tex:916]: Change to "The number of Active...".

File: constitution.tex [hunk 5]
```diff
@@ -907,95 +905,71 @@
...
-\asubsection{Total Number of Votes Cast}
-The Total Number of Votes Cast is defined as the total number of votes received for every voting option minus the number of abstentions.
+\asubsection{Votes Cast}
+The sum of the number of votes, excluding Abstentions.
```
Comment (Sam-Cordry) [inline constitution.tex:918]: "The number of votes cast, excluding Abstentions."

File: constitution.tex [hunk 6]
```diff
@@ -907,95 +905,71 @@
...
 \asubsection{Quorum}
-A Quorum is defined by the minimum number of votes cast required for a vote to be official.
-It is a fraction or a percentage of the total number of possible votes.
+The minimum Votes Cast for vote to be considered valid.
 Any member present for an Immediate Vote or given a ballot who does not explicitly cast their vote is counted as an Abstention.
-A Quorum is reached if the Total Number of Votes Cast plus the number of Abstentions is equal to or exceeding the minimum number of votes required.
+A quorum is expressed as a fraction or percentage of Eligible Votes (rounded up) unless otherwise specified.
```
Comment (Sam-Cordry) [inline constitution.tex:922]: "...Eligible Votes (rounded up to the nearest vote) unless..."

File: constitution.tex [hunk 7]
```diff
@@ -907,95 +905,71 @@
...
 \asubsection{Proxy Ballot}
```
Comment (rhochgraf21) [inline constitution.tex:923]: Proxy Vote

File: constitution.tex [hunk 8]
```diff
@@ -907,95 +905,71 @@
...
-A secret immediate vote lasts as long as it takes for all votes to be tallied.
+An Immediate Vote lasts as long as it takes for all votes to be tallied.
+\asubsubsection{Secret Immediate Vote}
+An Immediate Vote in which votes are kept anonymous.
+\asubsubsection{Executive Board Vote}
+In an Executive Board Vote, only Voting Members of the Executive Board are eligible to vote.
+Unless otherwise specified, Executive Board votes are Immediate Relative Majority votes with a fifty percent Quorum.
+
 \asubsection{Batch Vote}
 When a Batch Vote is called for by the Chair of the Vote, a subset of the voting docket may be amended to a single vote.
```
Comment (Sam-Cordry) [inline constitution.tex:960]: "Chair of the Vote, a non-empty subset"

File: constitution.tex [hunk 9]
```diff
@@ -907,95 +905,71 @@
...
+Approval Criteria are the rules that determine whether an option in a vote has passed.
+\asubsection{Relative Majority}
+In a Relative Majority vote, an option in the vote passes if the number of votes cast for that option is larger than the number of votes cast for every other option individually.
+\asubsection{Fractional}
+In a Fractional vote, an option in the vote passes if the number of votes cast for that option exceeds a specified fraction of the Votes Cast, rounded up.
+The fraction is specified in the text describing the vote (eg, "Two-Thirds Vote").
+\asubsection{Ranked Choice}
+In a Ranked Choice vote, voters rank the options by placing a '1' by their first choice, a '2' by their second choice, and so on, until they no longer wish to express any further preferences or run out of options.
 The winning option is selected outright if it gains more than half the votes cast as a first preference.
 If not, the option with the fewest number of first preference votes is eliminated and their votes move to the second preference marked on the ballot papers.
```
Comment (Sam-Cordry) [inline constitution.tex:974]: Nuke "papers".

File: constitution.tex [hunk 10]
```diff
@@ -1004,16 +978,15 @@
 \asubsection{With Pass/Fail}
 If the number of votes cast for the pass option equals the number of votes cast for the fail option, then the vote has failed.
 \asubsection{With Multiple Options}
-If multiple options may pass, a tie does not present a problem.
 If only one option may pass, then the vote must be recast or tabled at the discretion of the Chair of the Vote.
-In the event of a tie in an Executive Board vote, the Chairperson may cast the tie-breaking vote.
+In the event of a tie in an Executive Board Vote, the Chairperson may cast the tie-breaking vote.
```
Comment (Sam-Cordry) [inline constitution.tex:982]: "the Chairperson casts the tie-breaking vote."(?)

File: constitution.tex (hunk 1 above)
Comment (rhochgraf21) [reply constitution.tex:563]: fixed

File: constitution.tex (hunk 2 above)
Comment (rhochgraf21) [reply constitution.tex:822]: no, it is not.

File: constitution.tex (hunk 3 above)
Comment (rhochgraf21) [reply constitution.tex:838]: fixed.

File: constitution.tex (hunk 4 above)
Comment (rhochgraf21) [reply constitution.tex:916]: 👍

File: constitution.tex (hunk 5 above)
Comment (rhochgraf21) [reply constitution.tex:918]: fix'd

File: constitution.tex (hunk 6 above)
Comment (rhochgraf21) [reply constitution.tex:922]: 🥇

File: constitution.tex (hunk 7 above)
Comment (rhochgraf21) [reply constitution.tex:923]: 👍

File: constitution.tex [hunk 11]
```diff
@@ -907,95 +905,71 @@
...
 Proxy Votes are only permissible where explicitly stated, at the discretion of the Chair of the Vote.
+Proxy Votes must be documented in writing and signed by the absent member.
+The count of all Proxy Vote must be recorded and announced in all votes.
 \asubsection{Abstention}
-An Abstention is defined as a vote indicating a neutral position in the vote.
+An Abstention is a neutral vote that counts towards the Quorum but not towards determining the outcome of a vote.
 A means to abstain must always be provided in a vote.
-Abstentions are counted towards a Quorum, but not towards the Total Number of Votes Cast used to determine if a vote passes or not.
 \asubsection{Vote Counters}
 The Chair of the Vote is a vote counter and will additionally select two other members to count votes.
```
[the remark this answers is not in the document]
Comment (rhochgraf21) [reply constitution.tex:932]: 👍

**Comment by rhochgraf21** [comment]
ready for review

**Review by Sam-Cordry — APPROVED** [review]
After implementing feedback from discussion, this looks good to me.

File: constitution.tex [hunk 12]
```diff
@@ -692,7 +689,7 @@
 \asubsubsubsection{On-Floor Status}
 Members with On-Floor status are eligible to live on the floor.
 To be granted On-Floor status, members may notify the Evaluations Director that they would like to come up for a vote.
-The Evaluations Director will then bring them up for vote at the next meeting.
+The Evaluations Director will then bring them up for a One-Half Immediate vote with twenty-five percent Quorum at the next evaluations meeting.
```
[the remark this answers is not in the document]
Comment (esoccoli) [reply constitution.tex:692]: This should be changed to say that it is a vote among attending members. Also, mention something about how the evals director may decide there arent enough people present to vote

**Review by esoccoli — CHANGES_REQUESTED** [review]
For an on floor vote to, it should be changed to say that it is a vote among attending members. Also, mention something about how the evals director may decide there arent enough people present to vote

**Comment by Sam-Cordry** [comment]
@rhochgraf21 Here you go.

File: constitution.tex (hunk 12 above)
[the remark this answers is not in the document]
Comment (rhochgraf21) [reply constitution.tex:692]: changed 👍

**Comment by rhochgraf21** [comment]
@esoccoli updated with the changes you noted

**Review by esoccoli — APPROVED** [review]
lgtm

**Review by danisaba — APPROVED** [review]
Looks good to me.

**Pull request merged**